feat: add config file and use user's prompted input when writing to f…

Vulth-01 opened the pull request:
…iles

Vulth-01 committed:
feat: add config file and use user's prompted input when writing to files

paddatrapper commented on config.txt:
```diff
@@ -0,0 +1,35 @@
+- file: "J294.-.Death.Notice.pdf"
```
Remove this file

paddatrapper commented on pdfff/__main__.py:
```diff
@@ -28,14 +30,38 @@ def getInput():
             print("Please insert your ID:")
             index = 2
 
+        elif (index == 2) :
```
This is a very inefficient way of handling a lot of input. Rather do something like the following:

```python
questions = [
"Please insert Deceased ID:"
"Please insert  Deceased date of birth (DD/MM/YYYY):"
]
data = {}

for question in questions:
    answer = input(question)
    data[question] = answer
```

That way the only thing needed to add an additional question is to update the questions list. This makes extending things to support a configuration file much easier

paddatrapper commented on pdfff/config.yml:
```diff
@@ -0,0 +1,195 @@
+  fields:
+    - id: 'Text4-1'
+    - Full name: ''
+    - Full name 2: ''
```
`Full name 2` isn't very descriptive

Vulth-01 committed:
feat: add config file data in pyaml_env format

Issues with exporting/writing to yml file from main.

paddatrapper reviewed (changes requested):
Remove the files under test from the commits and add a line in `.gitignore` to ignore `tests/*`

Vulth-01 committed:
feat: pass user's input to config.yml file

Additional fields can be added, but this shows that the program is working overall.

Vulth-01 committed:
fix: change method of information passing

Project prompts user for input and passes info through to pdf files

paddatrapper reviewed (changes requested):
Please remove the files under OLD_TEST and test from this PR

paddatrapper committed:
fix: remove generated files

paddatrapper committed:
fix: ignore generated files

Vulth-01 committed:
fix: remove redundant field and change duplicated field

Vulth-01 committed:
fix: set fullname for questions
